packages/examples/simple-blog/blog/class-naming-strategies.md

```diff
@@ -8,17 +8,14 @@ credits: ui8
 
 In web development, CSS and design work hand-in-hand to create compelling user experiences. CSS brings designs to life digitally, translating visual mockups into actual rendered elements. But CSS also informs the design process itself. Understanding its capabilities and limitations allows designers to craft effective layouts and interfaces.
 
-
 ## Keep them short
 
 When designing for the web, keeping CSS in mind from the start prevents frustration down the road. For example, layered graphical elements must be broken down into achievable HTML components. Typography and color schemes are also heavily influenced by CSS possibilities.
 
-
 Designers should have a strong handle on core CSS concepts like the box model, selectors, specificity and inheritance.
 
 > When designing for the web, keeping CSS in mind from the start prevents frustration down the road. For example, **typography and color schemes are  heavily influenced by CSS** and layered graphical elements must be broken down into achievable HTML components.
 
```

Comment:
double space between "are heavily"

Reply:
Thanks. Fixed in https://github.com/nuejs/nue/commit/695111789427a09fe6f877f557159dbd066c1ac7